Univast/throttling.py

```diff
@@ -37,15 +51,14 @@ def get_cache_key(self, request: HttpRequest, view):
         client_scope, _ = self.get_client_scope_and_rate(api_key)
 
         self.scope = client_scope
-        self.get_rate()
+        self.get_rate(request)
 
         return self.cache_format % {
             "scope": client_scope,
             "ident": api_key,
         }
 
-    def get_rate(self):
-        request = self.request
```

Comment:
When you refactored this, did you test it?

Reply:
I haven't run tests yet, but I tested the API and accessing pages and they worked fine. 

I'm still on it.

Reply:
Alright.

Reply:
> Alright.

Tests passed, all good.